Backend/src/API/REST/routes/project/projectRouter.ts

```diff
@@ -54,17 +58,17 @@ const invite_person_to_a_project = (): Router => {
 
     router.put("/projects/:id/invite", check_auth(), async (req: any, res) => {
         try {
-            const {userId}: {userId: Schema.Types.ObjectId}  = req.body;
-            const projectId: {projectId: Schema.Types.ObjectId} = req.params.id;
+            const {userId}: {userId: ObjectId}  = req.body;
+            const {projectId}: {projectId: ObjectId} = req.params.id;
```

Comment:
Are these {} correct there? For userId they are correct, but for projectId?
```Javascript
//Lets assume
req.params.id = "abc";
req.body = {userId: "xyz", blabla: 123}; 
const {userId} = req.body; //Would result in userId = "xyz";
const {blabla} = req.body; //would result in blabla = 123;
const userId = req.body; //Would result in userId = {userId: "xyz", blabla:123};
const projectId = req.params.id; //Would result in projectId = "abc";
const {projectId} = req.params.id; //Results in I believe projectId = undefined;
```

Reply:
Oh I missed `params`. I thought it was in the body as well. But then it should be a string right? ``